fe1-web/src/features/evoting/components/ElectionOpened.tsx

```diff
@@ -146,7 +146,10 @@ const ElectionOpened = ({ election }: IPropTypes) => {
               </ListItem.Content>
             }
             onPress={() =>
-              setIsQuestionOpen({ ...isQuestionOpen, [question.id]: !isQuestionOpen[question.id] })
+              setIsQuestionOpen({
+                ...isQuestionOpen,
+                [question.id]: !isQuestionOpen[question.id],
+              })
```

Comment:
Sonar says that lines 138 to 154 are now an exact duplicate of code in ElectionQuestions.
Does it make sense to factor this out into its own component ? Just asking, not suggesting :)

Reply:
@Tyratox I'll let you fix this in a different PR ... if it's worth fixing. Be the judge of that 😁